swaglyrics_backend/issue_maker.py

```diff
@@ -542,12 +546,26 @@ def github_webhook():
             if payload['action'] == 'closed' and label == 'unsupported song' and repo == 'SwagLyrics-For-Spotify':
                 title = payload['issue']['title']
                 title = wdt.match(title)
-                song = title.group(1)
-                artist = title.group(2)
+                song, artist = title.groups()
                 logging.info(f'{song} by {artist} is to be deleted.')
                 cnt = del_line(song, artist)
                 return f'Deleted {cnt} instances from unsupported.txt'
 
+        elif event == "issue_comment":
+            # process comment commands
+            if payload['action'] != "created" and payload['repository']['name'] != 'SwagLyrics-For-Spotify':
+                return json.dumps({'msg': 'Not relevant issue comment create; ignoring'})
+
+            comment = payload['comment']
+            if comment['user']['id'] == 27063113 and comment['author_association'] == 'MEMBER':  # id of @aadibajpai
```

Comment:
If possible, let's avoid the hardcoding of the member ID.

Reply:
what would you suggest? since the ID is permanent, I felt it would be okay to not use a variable here—later if we want to extend it we can just remove that whole condition and the MEMBER check would be enough.

Reply:
Makes sense to me :)